modules/swagger-codegen/src/main/resources/python/api_client.mustache

```diff
@@ -142,14 +143,14 @@ class ApiClient(object):
         url = self.configuration.host + resource_path
 
         # perform request and return response
-        response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(method, url,
-                                     query_params=query_params,
-                                     headers=header_params,
-                                     post_params=post_params, body=body,
-                                     _preload_content=_preload_content,
-                                     _request_timeout=_request_timeout)
+        response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
+            method, url, query_params=query_params, headers=header_params,
+            post_params=post_params, body=body,
+            _preload_content=_preload_content,
+            _request_timeout=_request_timeout)
 
         self.last_response = response_data
+{{^tornado}}
```

Comment:
Resulted in a return statement within a generator which happens when tornado used.

Reply:
I don't understand it. Who calls `deserialize` ? This method returns `None` now.